### ISSUE TITLE
Missing documentation or functionality: cloning

### ISSUE THREAD

**Issue by kootenpv** [linode/linode_api4-python#127]
I cannot find in the documentation how I can clone.

I'm assuming it is being done the "old" way (I see the "images" tab in the new frontend).

1. create linode
2. clone a disk
3. boot linode

How to do step 2?

**Comment by kootenpv** [comment]
Already tried the new way. I felt like I got very close with creating the image:

```python
region = get_region("jp")
linode_type = get_linode_type("nanode")
client.linode.create_instance(linode_type, region, image=res)
```

It briefly showed up in my `client.get_images()` list, but then disappeared. I got a 400 error.

Could I please Beta test it or get a notification when I can use it?

**Comment by lgarber-akamai** [comment]
Addressed in #233. Feel free to @ me if this issue is still relevant 🙂
